samza-rest/src/main/java/org/apache/samza/monitor/MonitorConfig.java

```diff
@@ -96,4 +98,11 @@ public String getMonitorFactoryClass() {
   public int getSchedulingIntervalInMs() {
     return getInt(CONFIG_SCHEDULING_INTERVAL, DEFAULT_SCHEDULING_INTERVAL_IN_MS);
   }
+
+  public double getSchedulingJitterPercent() {
+    if (containsKey(CONFIG_SCHEDULING_JITTER_PERCENT)) {
+      return getDouble(CONFIG_SCHEDULING_JITTER_PERCENT) / 100.0;
+    }
+    return 0;
```

Comment:
Should be a constant like DEFAULT_SCHEDULING_INTERVAL_IN_MS.

Reply:
Fixed.